shared/types/CookieOptions.d.ts

```diff
@@ -1,5 +1,6 @@
 export interface CookieOptions {
   expires: Date;
   httpOnly: boolean;
+  sameSite?: "lax" | "none" | "strict" | undefined | boolean;
   secure?: boolean;
 }
```

Comment:
is it needed for it to be here? From FE side we don't see this as `CookieOptions` but as header in response, if I'm not mistaken

Reply:
I didn't really know where to put it. Of course location can be changed.